### DISCUSSION TITLE
Get song titles from Broadcasting Stream and not AutoDJ

### DISCUSSION THREAD

**Discussion by AndrewMD89** [AzuraCast/AzuraCast#6134] [Q&A] [answered]
Hi guys.

I'm new here and need a little help if possible.
My workflow is as follows: I stream from my PC to Azuracast into a mount point and get all the metadata (song title and artist name) from there. Works fine. I created a mount point with autodj and use it as fallback to my main stream. The problem is that once I start AutoDj, Azuracast reads the data from it (song and artist name). Is there a way to make Azuracast read the titles from the broadcast stream and not AutoDj? This issue ruins for me the song playback timeline report.
Thanks. :)

**Comment by Moonbase59** [comment, marked as answer]
Did you set up a "Streamer/DJ" account in AzuraCast and stream from your PC to _that_ mount point? Works flawlessly here, all Metadata intact and AutoDJ taking over smoothly. (I’m on Rolling Release #[29d12b7](https://github.com/AzuraCast/AzuraCast/commit/29d12b77bece3a25d8fc8f509afe6b5fc5ed3ab6) (2023-03-09 0:17).)

**Reply by AndrewMD89** [reply]
Thanks for the idea. I'll give it a try and come back with feedback.

**Reply by AndrewMD89** [reply]
Yes, works like a charm. Thanks again!

**Reply by Moonbase59** [reply]
You’re very welcome!
